class/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] The rentals variable should have `attr_accessor`. Kindly join it with the others.

  ```
  attr_accessor :title, :author, :rentals
  ```

Reply:
add rentals variable to attr_accessor